src/settings/lib/RuleEditor/CodeMirrorCustom.css

```diff
@@ -90,9 +90,9 @@
   width: 100%;
 }
 
+/* wrapper for the _entire_ list, not a single list-item */
 .CodeMirror-checkbox-container {
   width: 100%;
-  height: 12px;
```

Comment:
@zburke thank you 
I have one question
![image](https://user-images.githubusercontent.com/24813219/134163113-0b3fb97b-6f29-4281-8aa9-2decc9555be2.png)
Can we add style for fix focused/selected value? Just for example:
```
.CodeMirror-hints-list ul li {
  height: auto!important
}
```
![image](https://user-images.githubusercontent.com/24813219/134163491-95273906-234c-4ec1-9be0-2a8965c9225a.png)

Reply:
Nice find! I don't think we even need the `!important`, which I try to avoid as a rule. We can just change [the existing style on line 182](https://github.com/folio-org/ui-circulation/blob/fee7f30f799e0190d08e8f70d2a05f7630ed054f/src/settings/lib/RuleEditor/CodeMirrorCustom.css#L181-L183) from `12px` to `auto`.